radio/src/datastructs.h

```diff
@@ -911,6 +911,7 @@ PACK(struct TrainerData {
     uint8_t  slidersConfig:2; \
     uint32_t switchConfig; \
     uint8_t  potsConfig; /* two bits per pot */ \
+    NOBACKUP(uint8_t  blOffBright); \
```

Comment:
Would it be possible you steal 7bits from the bluetoothEnable so that we avoid changing again the EEPROM. I plan to add Json next week anyhow ...